aptos-move/framework/src/natives/aggregator_natives/aggregator_v2.rs

```diff
@@ -326,41 +395,33 @@ fn native_create_snapshot(
  **************************************************************************************************/
 
 fn native_copy_snapshot(
-    context: &mut SafeNativeContext,
-    ty_args: Vec<Type>,
-    mut args: VecDeque<Value>,
+    _context: &mut SafeNativeContext,
+    _ty_args: Vec<Type>,
+    _args: VecDeque<Value>,
 ) -> SafeNativeResult<SmallVec<[Value; 1]>> {
-    debug_assert_eq!(ty_args.len(), 1);
-    debug_assert_eq!(args.len(), 1);
-    context.charge(AGGREGATOR_V2_COPY_SNAPSHOT_BASE)?;
+    Err(SafeNativeError::Abort {
```

Comment:
Will `copy_snapshot` be supported in another PR?

Reply:
I am thinking of deciding when the rest of the implementation is there, and not supporting it in 1.8.

the framework move implementation doesn't use it any more. I would remove it , but it's signature is in 1.7